client/blocks/image-editor/test/image-editor-toolbar.jsx

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { shallow } from 'enzyme';
+import { expect } from 'chai';
+import { identity } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import useFakeDom from 'test/helpers/use-fake-dom';
+import { useSandbox } from 'test/helpers/use-sinon';
+
+describe( 'ImageEditorToolbar', () => {
+	useFakeDom();
+
+	let ImageEditorToolbar,
+		defaultProps,
+		wrapper;
+
+	useSandbox( ( sandbox ) => {
+		defaultProps = {
+			onShowNotice: sandbox.spy()
+		};
+	} );
+
+	before( () => {
+		ImageEditorToolbar = require( '../image-editor-toolbar' ).ImageEditorToolbar;
```

Comment:
I tried that first, but `useFakeDom()` needs to execute before the component is required. Any other order, or importing the component at the top, provokes:

`ReferenceError: window is not defined`

Without digging too deeply, I suspect one of the imported Popover components is attaching a click event to the document (for closing?).

I could mock these out, but it would mean more clutter in the test.